.github/workflows/build.yaml

```diff
@@ -52,11 +52,14 @@ jobs:
                   queries: +security-and-quality
 
             - name: Bootstrap
-              run: scripts/build/gn_bootstrap.sh
+              run: |
+                  eval `dmalloc -b -l DMALLOC_LOG -i 1 high`;
+                  export G_SLICE; G_SLICE=always-malloc; true required for dmalloc compatibility;
```

Comment:
true -> '#' ?

or ':' ?

Reply:
My editor implied that wouldn't work, but I'll try in the next push.

Reply:
Use colon if comments don't work, starting with true parses wrong.